linux-tkg-config/prepare

```diff
@@ -998,7 +998,7 @@ _tkg_srcprep() {
     _disable "MLX5_CORE"
   fi
 
-  if [ "$_eevdf_sched_ext_support" = "true" ] && [[ $_kver -ge 608 ]] || [[ $_kver -ge 612 ]]; then
+  if [ "$_eevdf_sched_ext_support" = "true" ] && [[ $_kver -ge 608 ]]; then
```

Comment:
Here I think we should keep the 
```
[ "$_eevdf_sched_ext_support" = "true" ] && [[ $_kver -ge 608 ]] || [[ $_kver -ge 612 ]]
``` 
and maybe even add parentheses
```
( [ "$_eevdf_sched_ext_support" = "true" ] && [[ $_kver -ge 608 ]] ) || [[ $_kver -ge 612 ]]
``` 
so it's always enabled for 6.12+